components/layout_2020/table/construct.rs

```diff
@@ -0,0 +1,509 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
```

Comment:
This is implementing https://html.spec.whatwg.org/multipage/tables.html#table-processing-model, right? Let’s have a link in a comment near the top of the file.